crates/viewer/re_space_view_spatial/src/visualizers/boxes3d.rs

```diff
@@ -64,8 +76,22 @@ impl Boxes3DVisualizer {
             // TODO(andreas): It would be nice to have this handle this fallback as part of the query.
             let radii =
                 process_radius_slice(entity_path, num_instances, data.radii, Radius::default());
-            let colors =
-                process_color_slice(ctx, self, num_instances, &annotation_infos, data.colors);
+            let surface_colors = process_color_slice(
+                ctx,
+                self,
+                num_instances,
+                // Don't use annotation-sourced colors for surface (only wireframe).
+                // TODO(kpreid): how *should* the annotation context interact with surface colors?
+                &ResolvedAnnotationInfos::Same(num_instances, ResolvedAnnotationInfo::default()),
+                data.solid_colors,
+            );
```

Comment:
Had the same thought. There's a lot of things not quite right with AnnotationContext and I agree for the very moment we'll just apply it only on the "old" colors.

Reply:
This is moot now that `SolidColor` doesn't exist.